distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSingleSignOn.java

```diff
@@ -359,15 +390,17 @@ public void testRedirectFlow() throws Exception {
         assertThat(idpHelper.parseBody(), is(Binding.REDIRECT));
         String inflatedSamlResponse = RestSecurity.inflateBase64(idpHelper.get("SAMLResponse"));
         validateSaml(inflatedSamlResponse, SamlSchema.PROTOCOL);
-        assertThat(inflatedSamlResponse, allOf(
-                containsString("urn:oasis:names:tc:SAML:2.0:status:Success"),
-                containsString("ds:SignatureValue"),
-                containsString("saml2:Assertion")));
-        assertThat(idpHelper.get("SigAlg"),    not(isEmptyOrNullString()));
+        assertThat(inflatedSamlResponse,
+                allOf(containsString("urn:oasis:names:tc:SAML:2.0:status:Success"),
+                        containsString("ds:SignatureValue"),
+                        containsString("saml2:Assertion")));
+        assertThat(idpHelper.get("SigAlg"), not(isEmptyOrNullString()));
         assertThat(idpHelper.get("Signature"), not(isEmptyOrNullString()));
-        assertThat(idpHelper.get("RelayState").length(), is(both(greaterThan(0)).and(lessThanOrEqualTo(80))));
+        assertThat(idpHelper.get("RelayState")
+                .length(), is(both(greaterThan(0)).and(lessThanOrEqualTo(80))));
```

Comment:
Only thing about this is, it is valid to have a blank relay state (we use a blank relay state in some of the logout stuff), So it probably works here as long as they're sending a relay state.. but in the case of no relay state it will be there but it will blank. So might want to do `greaterThanOrEqualTo(0)`.